locales/nl/base.json

```diff
@@ -0,0 +1,232 @@
+        "no-available-archives": "Geen archieven beschikbaar om naar te importeren."
+      },
+      "export": {
+        "export": "Exporteren",
+        "export-archive": "Exporteer {{name}} naar CSV",
+        "no-available-archives": "Geen archieven beschikbaar om te exporteren."
+      },
+      "close": "Sluiten"
+    },
+    "edit": {
+      "edit": "Bewerken",
+      "undo": "Ongedaan maken",
+      "redo": "Opnieuw",
+      "cut": "Knippen",
+      "copy": "Kopiëren",
+      "paste": "Plakken",
+      "pasteandmatchstyle": "Plak en combineer stijl",
+      "delete": "Verwijder",
+      "selectall": "Selecteer alles",
+      "speech": {
+        "speech": "Spreek",
+        "startspeaking": "Start spreken",
+        "stopspeaking": "Stop spreken"
+      }
+    },
+    "view": {
+      "view": "Beeld",
+      "condensed-sidebar": "Smalle zijbalk",
+      "language": "Taal",
+      "enable-tray-icon": "Geef weer in systeemiconen",
+      "reload": "Ververs",
+      "forcereload": "Geforceerd verversen",
+      "toggledevtools": "Dev tools in-/uitschakelen",
+      "togglefullscreen": "Full screen in-/uitschakelen",
+      "auto-hide-menubar": "Verberg menu automatisch"
+    },
+    "window": {
+      "window": "Venster",
+      "minimize": "Minimaliseren",
+      "close": "Sluiten",
+      "zoom": "Zoom",
+      "front": "Breng alles naar voorgrond"
+    },
+    "help": {
+      "help": "Help",
+      "visit-our-website": "Bezoek onze website",
+      "privacy-policy": "Privacy beleid",
+      "view-changelog-for-v": "Geef changelog weer voor v{{version}}"
+    },
+    "tray": {
+      "quit": "Afsluiten",
+      "open": "Open Buttercup"
+    }
+  },
+  "archive-search": {
+    "searchterm": "Zoeken...",
+    "nothing-found": "Niets gevonden"
+  },
+  "archive": {
+    "add-archive": "Voeg archief toe",
+    "archive-saved-loading-info": "Je archief wordt opgeslagen.<br>Dit venster sluit automatisch"
+  },
+  "archive-dialog": {
+    "buttercup-archives": "Buttercup archieven",
+    "load-a-buttercup-archive": "Laad een Buttercup archief",
+    "create-a-new-buttercup-archive": "Maak een nieuw Buttercup archief"
+  },
+  "archive-menu": {
+    "unlock": "Ontgrendelen",
+    "change-color": "Verander kleur",
+    "change-password": "Verander wachtwoord",
+    "import": "Importeren",
+    "import-from-type": "Van {{name}} archief (.{{extension}})",
+    "export": "Exporteer",
+    "archive-remove-with-name": "Verwijder {{name}}",
+    "open": "Open",
+    "lock": "Vergrendel"
+  },
+  "cloud-source": {
+    "cancel": "Annuleren",
+    "go-back": "Terug",
+    "new-archive": "Nieuw archief",
+    "open-in-buttercup": "Open in Buttercup",
+    "connect-to-dropbox": "Verbind met Dropbox",
+    "authenticate-with-dropbox": "Meld aan met Dropbox",
+    "dropbox-description-text": "Verbind Buttercup met Dropbox om je archieven te lezen en op te slaan.<br />We zullen je Dropbox gebruikersnaam en wachtwoord niet opslaan.",
```

Comment:
"Verbind Buttercup met Dropbox om je archieven te lezen en op te slaan.<br />We zullen je Dropbox gebruikersnaam en wachtwoord niet opslaan."

Perhaps the following is better:
"Verbind Buttercup met Dropbox om jouw archieven te lezen en op te slaan.<br />Dropbox gebruikersnaam en wachtwoord worden niet opgeslagen."